presto-main/src/main/java/com/facebook/presto/server/ServerConfig.java

```diff
@@ -38,6 +40,30 @@ public ServerConfig setCoordinator(boolean coordinator)
         return this;
     }
 
+    public boolean isEnabledUIonSecondaryPort()
+    {
+        return enabledUIonSecondaryPort;
+    }
+
+    @Config("http-server.ui.secondary.port.enabled")
```

Comment:
It's not clear that the example you show is achieving the same thing. It seems it could be disabling whether the HTTP port is disabled or not. In our case, what's the expected behavior when the UI port if different from the http port but `enabledUIonSecondaryPort` is false?